Test/Grains/ReminderGrain.cs

```diff
@@ -9,14 +9,14 @@ public sealed class ReminderGrain : Grain, IReminderGrain
     {
         public async Task RemoveReminder(string reminder)
         {
-            var reminderType = await GetReminder(reminder);
+            var reminderType = await this.GetReminder(reminder);
```

Comment:
There are a lot of these "this." fixes. I am fine when you change the coding style later, but for now we should revert this to make the review process easier.

Reply:
It's not coding style. They changed it from a base class method to an extension method, so the "this" is actually necessary.